packages/react/src/components/radio-button-group/radio-button-group.tsx

```diff
@@ -73,19 +80,31 @@ const StyledLabel = styled.label`
             `}
 `;
 
+const ContentWrapper = styled.div<{ isExpanded: boolean, height?: number }>(({ isExpanded, height = 500 }) => `
+  overflow: hidden;
+  max-height: ${isExpanded ? `${height}px` : '0'};
+  transition: ${isExpanded ? 'max-height 1s ease-in' : 'max-height .5s ease-out'};
+`);
+
+interface RadioButtonProps {
+    label: string;
+    value: string;
+    defaultChecked?: boolean;
+    disabled?: boolean;
+    content?: {
+        render: () => ReactElement;
```

Comment:
Je pense que je ferais plutôt
```suggestion
        element: React.ReactElement;
```
Je suis pas certain, mais je pense que React va avoir plus de faciliter à gérer son tree.

Reply:
J'avais tenté de faire ca de toute les façons possible et ca brisait toujours mon Storybook; Donc j'ai finalement choisi une function render